harvard-university-of-worcester.csl

```diff
@@ -261,14 +261,14 @@
       </else>
     </choose>
   </macro>
-  <citation collapse="year" disambiguate-add-year-suffix="true" et-al-use-first="3" et-al-min="4">
+  <citation et-al-min="3" et-al-use-first="1" sort-separator="," disambiguate-add-year-suffix="true" collapse="year" cite-group-delimiter=" ">
```

Comment:
Weird bug, but simple. It works correctly with class="note" on cs:style, but not with class="in-text". If that attribute should affect this at all (and I can't think of a reason why it should), that's backwards. Fix on its way ...